src/YoutubeAtomSticky.tsx

```diff
@@ -79,23 +79,29 @@ const stickyStyles = css`
     }
 `;
 
-const stickyContainerStyles = (height: number) => css`
-    height: ${height}px;
-    position: relative;
-    display: flex;
-    justify-content: flex-end;
+const stickyContainerStyles = (isMainMedia?: boolean) => {
+    const height = 192;
 
-    ${from.tablet} {
-        height: ${height * 2}px;
-    }
-`;
+    return css`
+        height: ${height}px;
+        position: relative;
+        display: flex;
+        justify-content: flex-end;
+        padding-right: ${isMainMedia ? '20px' : 0};
```

Comment:
Could we use `space[5]` from Source instead of `20px`?

Reply:
Yeah good spot, fixed.